### PR TITLE
{tools}[foss/2019b] Ray v0.8.4 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Darkless012** [comment]
This is build from wheel. Trying to build it from source.

**Comment by verdurin** [comment]
@Darkless012 any plans for how to handle the `Java` test failures?

**Comment by Darkless012** [comment]
Yup, I will include Java in version suffix since Bazel uses Java 1.8. 
But I'm, trying to build Ray from source, which is why I haven't pushed fix yet.

File: easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+source_urls = ['https://github.com/google/protobuf/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['83a3a0b23ed2b08041dbb0af92d023e5325a4324c2bc8acb55962d349710e52f']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('protobuf', version)
+]
+
+start_dir = 'python'
```
Comment (boegel) [inline easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2019b-Python-3.7.4.eb:21]: @Darkless012 Please also add:

```python
download_dep_fail = True
use_pip = True
```

File: easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2019b-Python-3.7.4.eb:26]: @Darkless012 Please also add:

```python
sanity_pip_check = True
```

File: easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (Darkless012) [reply easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2019b-Python-3.7.4.eb:21]: There are some issue with installing it via pip. Need to look closer.

Comment (Darkless012) [reply easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2019b-Python-3.7.4.eb:21]: installation fails with use_pip = True
the fosscuda 2018b version has it also disabled

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3416.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/af57cf1112560dbc5a00d9d709619e03 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3124.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/1f91a9606409d3d937bee43df19558cd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2606.swalot.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/c06a0221280ae0e29d02b47a36c3333d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/2f2b78a0a4e48adcd379a1bc4d5180ae for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**